dev/build/windows/makecoq_mingw.sh

```diff
@@ -119,7 +121,11 @@ mkdir -p "$PREFIXCOQ/bin"
 mkdir -p "$PREFIXOCAML/bin"
 
 # This is required for building addons and plugins
+# This must be CFMT (/cygdrive/c/...) otherwise coquelicot 3.0.2 configure fails.
+# coquelicot uses which ${COQBIN}/coqc to check if coqc exists. This does not work with COQBIN in MFMT.
 export COQBIN=$RESULT_INSTALLDIR_CFMT/bin/
+# This must be MFMT (C:/) otherwise bignums 68a7a3d7e0b21985913a6c3ee12067f4c5ac4e20 fails
+export COQLIB=$RESULT_INSTALLDIR_MFMT/lib/coq/
```

Comment:
I would rather be concerned about the CFMT. MFMT should be the default because paths might be sumbitted to MinGW binaries (liek coqc) which cannot handle CFMT paths.
I will try to fix this in coquelicot.

Reply:
Great!